jwst/assign_wcs/miri.py

```diff
@@ -69,9 +69,9 @@ def imaging(input_model, reference_files):
     try:
         bb = distortion.bounding_box
     except NotImplementedError:
-        shape = input_model.data.shape
+        shape = 1032,1024
```

Comment:
Hm, I see the complexity.  If we have a static shape=1032,1024 then the subarrays will not return errors even when a specified point is outside both the subarray and the full frame as per your example.  If we have a dynamic shape though, then the bounding box given also seems like it will be wrong for subarrays.  I.e., it will say that everything within 4 pixels of the left or right edge of the subarray doesn't map to the sky, which is untrue.  Is that right?  Some subarrays include reference pixels, and some do not.  It seems like we want the call to bounding_box to always be evaluated in full frame- does this make sense?